src/core_plugins/kibana/public/dashboard/panel/panel_utils.js

```diff
@@ -8,6 +9,12 @@ export class PanelUtils {
 
   // 6.1 switched from gridster to react grid. React grid uses different variables for tracking layout
   static convertPanelDataPre_6_1(panel) { // eslint-disable-line camelcase
+    ['col', 'row', 'size_x', 'size_y'].map(key => {
```

Comment:
It doesn't matter much, but this isn't really a map, as you aren't doing anything with the results, and you're using this as a side-effectful loop. I think this should be a `.forEach`